### PR TITLE
Update to metall 0.28

### PR DESCRIPTION
**Pull request by Clueliss**
Update to metall 0.28 and conan 2

### CODE REVIEW

File: tests/tests_Sanity.cpp [hunk 1]
```diff
@@ -19,7 +23,7 @@ TEST_SUITE("metall-ffi") {
             }
 
             {
-                auto *ptr = static_cast<size_t *>(metall_malloc(manager, obj_name, sizeof(size_t)));
```
Comment (bigerl) [inline tests/tests_Sanity.cpp:22]: why do you alloc twice the size here?

**Review by bigerl — CHANGES_REQUESTED** [review]
Thx for the PR. Just one question about the code.

File: tests/tests_Sanity.cpp (hunk 1 above)
Comment (bigerl) [reply tests/tests_Sanity.cpp:22]: yes!

Comment (bigerl) [reply tests/tests_Sanity.cpp:22]: If metall fails here to allocate a word we would also need to be always be very careful using it. 

This is blocker for updating metall.

Comment (Clueliss) [reply tests/tests_Sanity.cpp:22]: As I said, this has always been the case. Its not new

Comment (bigerl) [reply tests/tests_Sanity.cpp:22]: sorry, didn't read carefully. well then, we can merge, I guess.

**Review by bigerl — APPROVED** [review]

File: tests/tests_Sanity.cpp (hunk 1 above)
Comment (Clueliss) [reply tests/tests_Sanity.cpp]: As discussed I've reverted the manager settings to metall default, as they do not trigger this assertion

**Pull request merged**